popup.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable no-plusplus */
+
+// creation of popup container
+const popup = document.createElement('div');
+popup.setAttribute('id', 'popup');
+popup.setAttribute('class', 'popup');
+
+// Inserting popup container in document body
+const parentPop = document.body;
+parentPop.insertBefore(popup, parentPop.children[1]);
+popup.style.display = 'none';
+
+const mainProject = document.createElement('div');
+mainProject.setAttribute('class', 'popup-project');
+popup.appendChild(mainProject);
+
+const closeButton = document.createElement('div');
+closeButton.setAttribute('class', 'closex');
+closeButton.innerHTML = '&times;';
+mainProject.append(closeButton);
+
+const projectHeading = document.createElement('h1');
+projectHeading.setAttribute('class', 'popup-heading');
+mainProject.appendChild(projectHeading);
+projectHeading.innerText = 'Tonic';
+
+const projectSubDetails = document.createElement('p');
+projectSubDetails.setAttribute('class', 'popup-sub-details');
+mainProject.appendChild(projectSubDetails);
+projectSubDetails.innerText = 'CANOPY';
+
+const faIcon1 = document.createElement('i');
+faIcon1.setAttribute('class', 'circle fa-solid fa-circle');
+projectSubDetails.appendChild(faIcon1);
+
+const projectTitle = document.createElement('span');
+projectTitle.setAttribute = ('class', 'popup-title');
+projectSubDetails.appendChild(projectTitle);
+projectTitle.innerText = 'Back End Dev';
+
+const faIcon2 = document.createElement('i');
+faIcon2.setAttribute('class', 'circle fa-solid fa-circle');
+projectSubDetails.appendChild(faIcon2);
+
+const projectYear = document.createElement('span');
+projectYear.setAttribute('class', 'popup-year');
+projectSubDetails.appendChild(projectYear);
+projectYear.innerText = '2015';
+
+const projectImage = document.createElement('div');
+projectImage.setAttribute('class', 'popup-image');
+mainProject.appendChild(projectImage);
+
+// const projectImg1 = document.createElement('img');
+// projectImg1.setAttribute = ('alt', 'Project-2');
+// projectImg1.setAttribute = ('class', 'img-1');
+// projectImg1.src = './public/Snapshoot Portfolio.png';
+// projectImage.appendChild(projectImg1);
+
```

Comment:
- [ ]  Kindly consider removing all the old pieces of code commented. they make your project looks messy. If you need to review a previous version of your code, you can always use git history.